packages/@divvi/mobile/src/images/WelcomeLogo.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react'
 import { ViewStyle } from 'react-native'
 import Svg, { Path } from 'react-native-svg'
+import { getAppConfig } from 'src/appConfig'
 
 interface Props {
   width?: number
```

Comment:
Can we remove these props?
We're not using them right?

Reply:
we're not using them indeed but it's also not harmful to keep? I was trying to minimise changes / diffs with valora

Reply:
ah yes makes sense thanks!